pkg/Microsoft.ML.AutoMLFeaturizers/Microsoft.ML.AutoMLFeaturizers.nupkgproj

```diff
@@ -7,6 +7,8 @@
 
   <ItemGroup>
     <ProjectReference Include="../Microsoft.ML/Microsoft.ML.nupkgproj" />
+
+    <PackageReference Include="MicrosoftMLFeaturizers" Version="0.1.0" />
```

Comment:
Will this be the final package name of the native featurizers? If yes, then I find it confusing that the ML.NET package has the word `Auto` in it, but the underlying native package doesn't. What about the ML.NET package makes it "AutoML", but the underlying native package isn't "AutoML"?

Reply:
I will renaming the ML.NET package to be called NativeFeaturizers per Gani's suggestion. Or are you thinking since the native package is MicrosoftMLFeaturizers that I should name the ML.NET package the same thing?

---
In reply to: [326226514](https://github.com/dotnet/machinelearning/pull/4157#discussion_r326226514) [](ancestors = 326226514)

Reply:
Totally just my opinion, but I don't really like the word `Native` in the public API. I don't think it conveys any real information, and is an implementation detail.

It almost feels like the native assemblies, and the managed assemblies belong in the same NuGet package. But if we need to split them, I think their names should align in an understandable way.

Reply:
Sounds good to me. I'll rename the ML.NET package than to just be Microsoft.ML.Featurizers.

---
In reply to: [326238905](https://github.com/dotnet/machinelearning/pull/4157#discussion_r326238905) [](ancestors = 326238905)